tests/test_absolute.py

```diff
@@ -1,51 +1,47 @@
 import pytest
+from .helper import get_all_dtypes
 
 import dpnp as inp
 
 import numpy
 
 
-@pytest.mark.parametrize("type",
-                         [numpy.int64],
-                         ids=['int64'])
-def test_abs_int(type):
-    a = numpy.array([1, 0, 2, -3, -1, 2, 21, -9])
+@pytest.mark.parametrize(
+    "dtype", get_all_dtypes(no_bool=True, no_none=True, no_complex=True)
```

Comment:
I guess it might be helpful to check with default dtype also.
```suggestion
    "dtype", get_all_dtypes(no_bool=True, no_complex=True)
```